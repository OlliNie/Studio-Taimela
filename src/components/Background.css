.section{
  height: 100vh;
  text-align:center;
  background-image: url('../assets/images/city.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-attachment: fixed;
}
.ul{
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 0;
  height: 100%;
  margin: 0;
}
.li{
  width: 20vw;
  border-radius:100%;
  height: 20vw;
  background-color:#F1F1F0;
  color:black;
  display:flex;
  justify-content: center;
  align-items: center;
  margin:3rem;
  padding:1rem;
  opacity: 0.8;;
}
.content{
  padding-top:2rem;
  h1{
    letter-spacing: .7rem;
    font-weight: 100;
    font-size:3rem;
    color:black;
  }
}