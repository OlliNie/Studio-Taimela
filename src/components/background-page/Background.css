.background{
  width:auto;
  background-color:#F7F6F7;
  opacity: .9;
  display:flex;
  flex-direction: column;
  width: 200px;
  justify-content: center;
  padding: 2rem;
  @media(min-width: 750px){
    width:300px;
    margin-left:auto;
    margin-right:auto;
    margin-top:auto;
  }
  @media(min-width: 1100px){
    width:736px;
  }
}
.ulEducation{
  padding-left: 0;
  display: flex;
  flex-direction:column;
  align-items: center;
  li:first-child{
    font-weight:700;
  }
  li{
    font-weight:100;
  }
}
.skills{
  display:flex;
  flex-direction:column;
  justify-content: center;
  text-align: left;
  ul{
    padding:0;
    margin:0;
  }
  h2{
    margin:0;
    font-size:1rem;
  }
}