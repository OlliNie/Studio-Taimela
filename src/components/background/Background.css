.section{
  width:100%;
  min-height: 100vh;
  display:flex;
  flex-direction: column;
  text-align:center;
  background-image: url('../../assets/images/city.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-attachment: fixed;

}
.ulEducation{
  padding-left: 0;
  display: flex;
  flex-direction:column;
  align-items: center;
  color:black;
 
  li:first-child{
    font-weight:700;
  
  }
  li{
    font-weight:100;
  }

}

.background{
  background-color:#F7F6F7;
  opacity: .9;
  margin: 0 1rem;
  padding:1rem;
  color:black;
}
.skills{
  display:flex;
  flex-direction:column;
  @media (min-width: 720px){
    flex-direction: row;
  }
  justify-content: center;
  text-align: left;
  ul{
    padding:0;
    margin:0;
  }

  
  h2{
    margin:0;
    font-size:1rem;
  }
}
