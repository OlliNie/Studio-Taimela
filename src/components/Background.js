import React from 'react';
import styles from './Background.css';
import { background } from '../text/background';

export default function Background(){
  const backgroundItems = background.map((background, i)=>{

    return <li className={styles.li} key={i}><p>{background}</p></li>;
  });
  return (
    <section id='Background' className={styles.section}>
      <h1>Background</h1>
      <section className={styles.content}>
        <ul className = {styles.ul}>
          {backgroundItems}
        </ul>
      </section>

   

    </section>
  );
}
