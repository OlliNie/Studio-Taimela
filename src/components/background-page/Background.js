import React from 'react';
import Title from '../title/Title';
import Skills from './Skills';
import styles from './Background.css';
import { education } from '../../text/background';

export default function Background() {
  return (
    <section className={styles.background}>
      <Title title="background" />
      <section className={styles.backgroundText}>
        <section>
          <ul className={styles.ulEducation}>
            <li>{education.degree}</li>
            <li>{education.school}</li>
          </ul>
        </section>
        <section className={styles.skills}>
          <Skills />
        </section>
      </section>
    </section>
  );
}
