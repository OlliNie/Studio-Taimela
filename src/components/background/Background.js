import React from 'react';
import styles from './Background.css';
import { education, languages, certificates } from '../../text/background';
import Title from '../title/Title';
import SkillMapper from './SkillMapper';
import { FaLinkedin } from 'react-icons/fa';

export default function Background() {

  return (
    <section id='Background' className={[styles.backgroundPage, 'backgroundPage'].join(' ')}>
    
      <section className={styles.background}>
        <Title title='background' />

        <section className={styles.backgroundText}>
          <section>
            <ul className={styles.ulEducation}>
              <li>{education.degree}</li>
              <li>{education.school}</li>
            </ul>
          </section>
          <section className={styles.skills}>
            <SkillMapper title={languages.title} skills={languages.skills} />
            <SkillMapper title={certificates.title} skills={certificates.skills} />
          </section>
        </section>

      </section>
      <section className={styles.social}>
        <p>
            Find my detailed work experience on LinkedIn
        </p>
        <a alt="Linked In" aria-label= "Linked In" rel= "noopener noreferrer" target = '_blank' href='https://www.linkedin.com/in/virpinieminen/'>
          <FaLinkedin size={42} />
        </a>
      </section>



    </section>
  );
}
