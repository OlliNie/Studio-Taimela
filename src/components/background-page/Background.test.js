import React from 'react';
import { shallow } from 'enzyme';
import Background from './Background';

describe('Background component', () => {
  it('renders Background', () => {
    const wrapper = shallow(<Background />);
    expect(wrapper).toMatchSnapshot();
  });
});
