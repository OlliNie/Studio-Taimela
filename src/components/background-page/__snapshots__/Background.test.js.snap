// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background component renders Background 1`] = `
<section
  className="background"
>
  <Title
    title="background"
  />
  <section
    className="backgroundText"
  >
    <section>
      <ul
        className="ulEducation"
      >
        <li>
          Master's degree in Management Science
        </li>
        <li>
          Solvay Brussels Schoool, Belgium
        </li>
      </ul>
    </section>
    <section
      className="skills"
    >
      <Skills />
    </section>
  </section>
</section>
`;
